ibis_substrait/compiler/translate.py

```diff
@@ -634,6 +634,31 @@ def _count(
     )
 
 
+@translate.register(ops.StandardDev)
+@translate.register(ops.Variance)
+def _variance_base(
+    op: ops.StandardDev | ops.Variance,
+    expr: ir.ValueExpr | None = None,
+    *,
+    compiler: SubstraitCompiler | None = None,
+    **kwargs: Any,
+) -> stalg.AggregateFunction:
+    if compiler is None:
+        raise ValueError
+    if op.how == "pop":
+        raise NotImplementedError("`how='pop'` not yet implemented")
+    translated_arg = stalg.FunctionArgument(
+        value=translate(op.arg.op(), compiler=compiler, **kwargs)
+    )
+    return stalg.AggregateFunction(
+        function_reference=compiler.function_id(op=op),
+        arguments=[translated_arg],
+        sorts=[],  # TODO: ibis doesn't support this yet
```

Comment:
@gforsyth If I wanted to hard-code something here to set the `distribution` option to `SAMPLE` in the plan, how would I do that?
```py
options = ???
```

Reply:
I think Sth like
```
options = stalg.FunctionOption(name='distribution', preference=['SAMPLE'])
```

Reply:
On mobile but I believe Li is correct.

Note that since 'sample' is the first option in the yaml files it will be chosen by default if nothing is set (or so says substrait convention)

Reply:
Thanks, yes that worked. I implemented support for `how="pop"` in 589c882.